README.md

```diff
@@ -1,4 +1,11 @@
 # Load Watcher
 
-The load watcher is responsible for the cluster-wide aggregation of resource usage metrics like CPU, memory, network, and IO stats over time windows from a metrics provider like SignalFx, Prometheus, K8s etc.
-It stores them in its local cache, which can be queried from scheduler plugins.
\ No newline at end of file
+The load watcher is responsible for the cluster-wide aggregation of resource usage metrics like CPU, memory, network, and IO stats over time windows from a metrics provider like SignalFx, Prometheus, Kubernetes etc. developed for [Trimaran: Real Load Aware Scheduling](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/kep/61-Trimaran-real-load-aware-scheduling/README.md) in Kubernetes.
```

Comment:
I think you are referring to the Kubernetes metrics-server when you mention Kubernetes.

Reply:
from metric providers like SignalFx, Prometheus, Kubernetes metrics-server, etc.

Reply:
We also have support for prometheus coming in right?

Reply:
Yes @wangchen615 is working on it.